storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3257,6 +3257,17 @@ std::string dump_ingest_external_file_options(
   return s.str();
 }
 
+/** Set of ids of DD tables */
```

Comment:
Should this whole block go to i.e. `rdb_datadic.h`? Actually we could start even a new source file `rdb_native_dd` or similar.

Reply:
starting a new source file sounds good to me. We will have a bunch of dd related variables and Innodb also has it's own dd related source file.